### ISSUE TITLE
Site is not intended to be a fixed width

### ISSUE THREAD

**Issue by farishkash** [valconix/f1-3-c2p1-colmar-academy#1]
So the idea with Colmar is it is suppose to be a large full width site.  You set a width of 960px in your main container.  So if you are on a laptop you may not see or notice this, but this causes the site to be much smaller than intended.  Screenshot with your 960px width.

![colmar_academy960](https://user-images.githubusercontent.com/5844161/30235833-a826efc4-94c2-11e7-981f-5838e7a50f10.jpg)

And here is what it looks like when I uncheck this in the chrome dev tools.  The chrome dev tools were open in the first shot and I didn't change the screen resolution.

![colmar_academy](https://user-images.githubusercontent.com/5844161/30235845-ee91b75a-94c2-11e7-899c-f6812f7f7592.jpg)

Other than this there is no "flaw" in your website it is really well done.  Will talk about other practices in my summary.

**Comment by valconix** [comment]
Yes, I didn't like the look of it when it was expanded, somehow the button's weren't 'in proportion' to everything, especially when playing with the width, so I added the 960 (and now I'm reading all about bootstrap and thinking! darn it, that's exactly what would have made things easier and cleaner! Thanks for the review!
